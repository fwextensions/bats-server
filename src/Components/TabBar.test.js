import React from 'react';
import renderer from 'react-test-renderer';
import TabBar from './TabBar';

test('Snapshot', () => {
  const component = renderer.create(
    <TabBar
      onSelect={() => {}}
      selectedTab={1}
      tabs={['Ringdown', 'Hospital Info']}
    />
  );

    const componentJson = component.toJSON();
    expect(componentJson).toMatchSnapshot();
});