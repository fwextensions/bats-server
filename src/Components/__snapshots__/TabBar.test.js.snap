// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot 1`] = `
<div
  className="tabbar"
>
  <button
    className="tabbar__tab h4"
    onClick={[Function]}
    type="button"
  >
    Ringdown
  </button>
  <button
    className="tabbar__tab tabbar__tab--selected h4"
    onClick={[Function]}
    type="button"
  >
    Hospital Info
  </button>
</div>
`;
